Dockerfile

```
@@ -168,6 +109,8 @@ RUN echo "Software installed:"; \
     infracost --version; \
     echo "Prettier v$(prettier --version)"; \
     echo "Regula $(regula version)"; \
-    echo "Bun v$(bun --version)"
+    echo "Bun v$(bun --version)"; \
+    terraform --version; \
```

```suggestion
    echo "Terraform v$(terraform version -json | jq -r '.terraform_version')" \
```

note: we might need to install `jq` - it probably is already installed though. and if not we probably should. it's always useful to have around.

---

jq is already there :) https://github.com/Stabl-Energy/spacelift-runner-cdktf/blob/5f3dd7f25e4f51674b947fee03c2d170140de26e/Dockerfile#L36 but thanks for the tip!